app/qml/MerginProjectPanel.qml

```diff
@@ -463,6 +463,24 @@ Item {
                 } else __inputUtils.showNotification(qsTr("No Changes"))
               }
             }
+
+            MenuItem {
+              height: contextMenu.isMerginProject ? 0: contextMenu.menuItemHeight
+              ExtendedMenuItem {
+                  height: parent.height
+                  rowHeight: parent.height
+                  width: parent.width
+                  contentText: contextMenu.isMerginProject ? qsTr("Detach from Mergin") : qsTr("Upload to Mergin")
+                  imageSource: contextMenu.isMerginProject ? InputStyle.detachIcon : InputStyle.uploadIcon
```

Comment:
Better to remove the "Detach from Mergin" text and the detach icon to avoid confusion?